client/templates/orderDetails/status/orderPacked/orderPacked.html

```diff
@@ -23,30 +23,20 @@
         <h2>Info for Ship Station</h2>
         <h3>Order #</h3>
         <h3><strong>{{displayOrderNumber this}}</strong></h3>
-        <br>
-        <br>
+        <a href='https://ss5.shipstation.com/#/orders' id='labelPrint' target='_blank' class='btn btn-block btn-lg btn-default'>Print Shipstation Label</a>
```

Comment:
is the `ss5.` subdomain a loadbalancer that we shouldn't be linking to or is that our specific server?